report/src/main/webapp/app/layout/sidepanel/sidepanelResizer.directive.js

```diff
@@ -0,0 +1,72 @@
+/*
+ * AET
+ *
+ * Copyright (C) 2013 Cognifide Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+define(['angularAMD'], function (angularAMD) {
+  'use strict';
+  angularAMD.directive('aetSidepanelResizer', SidepanelResizerDirective);
+
+  function SidepanelResizerDirective() {
+    return {
+      restrict: 'AE',
+      scope: {
+        'type': '@'
+      },
+      link: linkFunc
+    };
+
+    function linkFunc($scope, $element) {
+      var $body = $('body');
+      var $content = $('.main');
```

Comment:
Is there a better way of getting that elements? Can't the directive be attached to higher element in DOM and use elements in it's scope?